packages/vscode-container-client/package.json

```diff
@@ -24,7 +24,8 @@
     },
     "dependencies": {
         "dayjs": "^1.11.2",
-        "tree-kill": "^1.2.2"
+        "tree-kill": "^1.2.2",
+        "zod": "^3.24.2"

```

Comment:
It may be worth upgrading to zod 3.25 to get the new zod 4 release. Looks to be backwards compatible with the types you're currently implementing, but with some new features around data coercion.

Reply:
Those errors are fixed, upgrade to 3.25.56 👍